stdlib/xml/dom/minidom.pyi

```diff
@@ -290,47 +293,47 @@ class Document(Node, DocumentLS):
     previousSibling: Any
     nextSibling: Any
     actualEncoding: Any
-    encoding: Any
-    standalone: Any
+    encoding: str | None
+    standalone: bool | None
     version: Any
     strictErrorChecking: bool
     errorHandler: Any
     documentURI: Any
-    doctype: Any
+    doctype: DocumentType | None
     childNodes: Any
     def __init__(self) -> None: ...
     def appendChild(self, node): ...
     documentElement: Any
     def removeChild(self, oldChild): ...
     def unlink(self) -> None: ...
     def cloneNode(self, deep): ...
-    def createDocumentFragment(self): ...
-    def createElement(self, tagName: str): ...
-    def createTextNode(self, data): ...
-    def createCDATASection(self, data): ...
-    def createComment(self, data): ...
+    def createDocumentFragment(self) -> DocumentFragment: ...
+    def createElement(self, tagName: str) -> Element: ...
+    def createTextNode(self, data: str) -> Text: ...
+    def createCDATASection(self, data: str) -> CDATASection: ...
+    def createComment(self, data: Incomplete) -> Comment: ...
```

Comment:
Should be `str`. It gets passed to `Comment` and `Comment.writexml` assumes its data is a str: https://github.com/python/cpython/blob/18b1782192f85bd26db89f5bc850f8bee4247c1a/Lib/xml/dom/minidom.py#L1208